metrics/failures-config.json

```diff
@@ -0,0 +1 @@
+{"query": "#standardSQL\nselect /* Find jobs that have not passed in a long time */\n  jobs.job,\n  latest_pass, /* how recently did this job pass */\n  weekly_builds,  /* how many times a week does it run */\n  first_run, /* when is the first time it ran */\n  latest_run,  /* when is the most recent run */\n  DATE_DIFF(current_date(), if(latest_pass is null, first_run, date(latest_pass)), DAY) broken_days\nfrom (\n  select /* filter to jobs that ran this week */\n    job,\n    count(1) weekly_builds\n  from `k8s-gubernator.build.all`\n  where\n    started > timestamp_sub(current_timestamp(), interval 7 day)\n  group by job\n  order by job\n) jobs\nleft join (\n  select /* find the oldest, newest run of each job */\n    job,\n    date(min(started)) first_run,\n    date(max(started)) latest_run\n  from `k8s-gubernator.build.all`\n  group by job\n) runs\non jobs.job = runs.job\nleft join (\n  select /* find the most recent time each job passed (may not be this week) */\n    job,\n    max(started) latest_pass\n  from `k8s-gubernator.build.all`\n  where\n    result = 'SUCCESS'\n  group by job\n) passes\non jobs.job = passes.job\nwhere\n  latest_pass is null\n  or DATE_DIFF(current_date(), date(latest_pass), day) > 30\norder by broken_days desc, latest_pass, first_run, weekly_builds desc, jobs.job\n", "metric": "failures", "jqfilter": "[(.[] | {(.job): {\n      failing_days: (.broken_days)\n  }})] | add"}
```

Comment:
Can you make this file YAML instead? This file is basically unreadable, and diffing it will be annoying. Yaml has [nice multiline strings](https://til.hashrocket.com/posts/d7c96e2ee7-multiline-strings-in-yaml).

You should be able to install python-yaml in the Docker image.